### ISSUE TITLE
Supporting IE11

### ISSUE THREAD

**Issue by gharibi** [gharibi/JsObjExporter#3]
Currently `object-exporter` supports the export to CSV and XLS for Chrome version 71.0.3578.98, Safari version 12.0.2 (14606.3.4), FireFox version 64.0 and Opera version 57.0.3098.76. **Let's say I have tested the library on the above platforms.**

However IE11 is not supported for the both XLS and CSV exports. I had a quick look at the `xls.js` and the issue seem to be from here:
```
  let ctx = { worksheet: sheetName, table: dataset }
  let a = document.createElement('a')
  a.href = uri + base64(format(template, ctx))
  a.download = fileName + '.xls'
  a.click()
```

Maybe there are alternatives to the above code. Alternatively `window.location` could be used instead of `a.click()`, but the problem is, then the `fileName` cannot be set and moreover it will cause a redirection on the page.

**Comment by gharibi** [comment]
Looking at this post from microsoft team in [HERE](https://social.msdn.microsoft.com/Forums/Lync/en-US/2701cd0e-cbb8-454b-be65-3b4376b653d5/download-base64-data-ie11?forum=iewebdevelopment), I think we can never support IE.

According to what they say: 

> PDF and Excel documents(+Word, powerpoint etc) (regardless of binary encoding) are handled by Activex controls in MSIE<11 browsers... not natively, so you cannot just assign the data uri to the windows' href property. the windows or frames collections cannot 'load' non-html content... it is rather hosted in an ActiveX control that is 'framed' in the window,iframe, object or embed. 

> you should use the object tag and dynamically change both the data attribute and the type attributes according to the mime-type of the data-uri.

> IE11 has 'ActiveX Filtering' which can affect the outcomes on clients hosting ActiveX loaded content.

Well, I am not sure what else could be done?!

**Comment by gharibi** [comment]
This issue is solved as part of v3.6.1.
